content/docs/2.14/scalers/mongodb.md

```diff
@@ -92,6 +95,7 @@ As an alternative to environment variables, You can authenticate with the MongoD
 
 **Password Authentication:**
 
+- `scheme` - The scheme of the MongoDB server, if using MongoDB Atlas, you can set it to `mongodb+srv`. (Default: `mongodb`)
```

Comment:
We have to add also that's optional (https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler)
```suggestion
- `scheme` - The scheme of the MongoDB server, if using MongoDB Atlas, you can set it to `mongodb+srv`. (Default: `mongodb`, Optional)
```